docs/src/environments/backup.md

```diff
@@ -9,28 +9,40 @@ you want to be sure those changes only improve things and don't make you lose an
 You also want to have a disaster recovery plan in place.
 Backups help you protect yourself against potential data loss.
 
-{{< note theme="warning">}}
-
-Backups include the environment's complete data but exclude code.
-To restore code to its previous state when the backup was taken,
-use Git commands such as [revert](https://git-scm.com/docs/git-revert).
-
-{{</note>}}
-
-Backups include all persistent data from all running [services](../add-services/_index.md)
-and any files stored on [mounts](../create-apps/app-reference.md#mounts).
-The snapshot is stored internally and can't be downloaded.
-
 You might want to create backups of your live environment before merging a different environment into it
 or each time you increase the storage space of your services.
 
 You also have regularly scheduled automated backups of your production environments to cover most cases.
 
-You can only create backups and restore active environments.
+Note that you can only create backups and restore active environments.
 To work with an [inactive environment](../other/glossary.md#inactive-environment),
 first activate it.
 
-To create a backup of an environment, you need an [Admin role for that environment type](../administration/users.md).
+## How backup and restore works on Platform.sh
+
+1. As an [admin user](../administration/users.md), you can do a backup of your environment. 
+   This backup includes the complete data and code of the environment.
+   All persistent data from all running [services](../add-services/_index.md)
+   and any files stored on [mounts](../create-apps/app-reference.md#mounts) are included.
+   The snapshot is stored internally and can't be downloaded.
```

Comment:
```suggestion
   The snapshot is stored internally on Platform.sh.
   That is, the backup can be applied to environments on Platform.sh, but it can't be downloaded.
   If you need to download backups, instead [export your mount and service data](https://docs.platform.sh/tutorials/exporting.html).
```

Reply:
Also, I know that `snapshot` was what we used to call this feature, which was switched to backups - even though the `snapshot` command technically still works in the CLI.

How do we feel about word choice consistency on this page?